notebooks/notebooks/iclr2022/estimate-flops-katago.ipynb

```diff
@@ -0,0 +1,662 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #23.    sys.path.append(str(GIT_ROOT / "submodules/KataGo-pytorch-rewrite/python"))
</pre><p>Modifying <code>sys.path</code> feels a bit fragile. I was going to suggest just <code>pip install -e</code> this directory but I guess KataGo doesn't have a <code>setup.py</code> so you can't actually do that, yuck. I'd suggest at least adding a comment explaining why this is the least-bad option.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HumanCompatibleAI/KataGoVisualizer/pull/40/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Added a comment explaining why it is the least bad option.